src/Components/Button/Button.stories.jsx

```diff
@@ -0,0 +1,26 @@
+import { React } from 'react'
+import Button from './Button'
+
+export default {
+    title: 'Button component',
+    component: Button
+}
+
+const Template = (args) => {
+    const {
+        className, onClick, children, ariaLabel
+    } = args
+    return (
+        <div>
+            <Button className={className} onClick={onClick} ariaLabel={ariaLabel}>{children}</Button>
```

Comment:
If prop and variable name is the same, usually we are using this syntax: 
`            <Button
                {...{
                    className,
                    onClick,
                    ariaLabel
                }}
            >
                {children}
            </Button>`